src/templates/import.js

```diff
@@ -684,6 +684,50 @@ module.exports = (router) => {
     });
   };
 
+  const tryToLoadComponent = (component, template) => {
+    const query = {
+      name: component.form,
+      deleted: {$eq: null}
+    };
+    return formio.resources.form.model.find(query).exec().then((results)=>{
+      if (results.length!==1) {
+       throw 'Incorrect project template';
+      }
+      const result = results[0];
+
+      const newItem = {
+          "title": result.title ,
+          "type": result.type,
+          "name": result.name,
+          "path": result.path,
+          "tags": result.tags,
+          "components": result.components
+      };
+
+      if (result.type==='form') {
+        template.forms[newItem.name]=newItem;
```

Comment:
Also, it won't work for some cases properly (and customers such as PWC will still have an issue), because the child form will come after the form it is used in. And if it revisions are allowed for the parent form, it will always  create a new revision of the parent form with wrong version of the nested form ('form' property is set to form's name). And as a result, we will have a correct form inside Form tabel, but a wrong version inside Formrevision tabel.